### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by gdelavald**
Add shitty Nodes dashboard, move cluster selection to drawer, fix bug in select cluster reducer

### CODE REVIEW

File: frontend/src/NodesDashboard.jsx [hunk 1]
```diff
@@ -1,15 +1,99 @@
...
+
+const mapStateToProps = ({ nodes }) => ({
+    nodes,
+});
+
+const mapDispatchToProps = {
+    loadNodes,
+}
 
 class ShardsDashboard extends React.Component {
```
Comment (chrissinclairedited) [inline frontend/src/NodesDashboard.jsx]: We should probably rename this

File: frontend/src/data/clusters/reducer.js [hunk 2]
```diff
@@ -33,7 +33,7 @@ export function reducer(state, action) {
         case CLUSTER_ACTION_TYPES.SET_CURRENT_CLUSTER:
             return {
                 ...state,
-                currentCluster: action.currentCluster,
+                currentCluster: action.clusterSlug,
```
Comment (chrissinclairedited) [inline frontend/src/data/clusters/reducer.js]: Have we changed the action creator to match this? Not sure if I'm just being blind but I can't see it?

Comment (gdelavald) [reply frontend/src/data/clusters/reducer.js]: I think this was the error that was messing up the Cluster selection, we were passing down clusterSlug but expecting currentCluster.

File: frontend/src/NodesDashboard.jsx (hunk 1 above)
Comment (gdelavald) [reply frontend/src/NodesDashboard.jsx]: Any ideas? We could prob sit down and try to figure out all the elements so the naming makes sense.

Comment (chrissinclairedited) [reply frontend/src/NodesDashboard.jsx]: `NodesDashboard` maybe? Yeah now we're not on the hackday definitely worth reassessing where everything is.

File: frontend/src/data/clusters/reducer.js (hunk 2 above)
Comment (chrissinclairedited) [reply frontend/src/data/clusters/reducer.js]: Ah yeah - looks like the clusters action creators are inconsistent:

https://github.com/EDITD/ElasticSurgery/blob/master/frontend/src/data/clusters/actions.js#L11 calls it `clusterSlug`, whilst https://github.com/EDITD/ElasticSurgery/blob/master/frontend/src/data/clusters/actions.js#L33 calls it `currentCluster`. That definitely needs fixed!

**Comment by Fizzadar** [comment]
@gdelavald @chrissinclairedited I'm gonna merge this and sort out the pending issues in another PR along with the issues in the other PR.

**Pull request merged**